config/settings_schema.json

```diff
@@ -159,7 +159,7 @@
         "default": "This website uses cookies"
       },
       {
-        "type": "contentEditor",
+        "type": "textarea",
```

Comment:
This should have been a `textarea` from the start. It was a confusion from my part.